src/yunohost/tests/test_changeurl.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+import time
+import requests
+
+from moulinette.core import init_authenticator
+from yunohost.app import app_install, app_changeurl, app_remove, app_map
+from yunohost.domain import _get_maindomain
+
+from moulinette.core import MoulinetteError
+
+# Instantiate LDAP Authenticator
+AUTH_IDENTIFIER = ('ldap', 'ldap-anonymous')
+AUTH_PARAMETERS = {'uri': 'ldap://localhost:389', 'base_dn': 'dc=yunohost,dc=org'}
+
+auth = init_authenticator(AUTH_IDENTIFIER, AUTH_PARAMETERS)
+
+# Get main domain
+maindomain = _get_maindomain()
+
+
+def setup_function(function):
+    pass
+
+
+def teardown_function(function):
+    app_remove(auth, "change_url_app")
+
+
+def install_changeurl_app(path):
+    app_install(auth, "./tests/apps/change_url_app_ynh",
```

Comment:
Not sure to understand the point ? You mean use [ynh_normalize_url_path](https://github.com/YunoHost/yunohost/pull/234) ? We already normalize the path from inside the python code [here](https://github.com/YunoHost/yunohost/blob/2b82a2e74dc625680459c4ce5825f19c2a49374b/src/yunohost/app.py#L367) and there are plans to use [this python helper](https://github.com/YunoHost/yunohost/blob/5125c0466a52ef8fe7ab2e2fd563f71b153f7474/src/yunohost/tools.py#L595) from PR #267 once it'll be available.